signup.go

```diff
@@ -241,12 +252,14 @@ func (su Signup) shortMessagingURL() (string, error) {
 	line1, cityStateZip := greenlight.ParseAddress(su.GooglePlace.Address)
 
 	p := rendererReqParams{
-		Template:     INFO_SESSION_TEMPLATE,
-		ZoomLink:     su.zoomMeetingURL,
-		Date:         su.StartDateTime,
-		Name:         su.NameFirst,
-		LocationType: su.LocationType,
-		JoinCode:     su.JoinCode,
+		Template:      INFO_SESSION_TEMPLATE,
+		ZoomLink:      su.zoomMeetingURL,
+		Date:          su.StartDateTime,
+		Name:          su.NameFirst,
+		LocationType:  su.LocationType,
+		JoinCode:      su.JoinCode,
+		IsGmail:       strings.HasSuffix(su.Email, "gmail.com"),
+		GreenlightURL: fmt.Sprintf("https://greenlight.operationspark.org/sessions/%s/?subview=overview&userJoinCode=%s", su.SessionID, su.userJoinCode),
```

Comment:
We should probably use an env var so that we can create URLs in any environment (local, testing, etc). We can add a parameter to this func.
```suggestion
		GreenlightURL: fmt.Sprintf("%s/sessions/%s/?subview=overview&userJoinCode=%s", greenlightHost, su.SessionID, su.userJoinCode),
```